docs/log-correlation.asciidoc

```diff
@@ -0,0 +1,96 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/ruby[elastic.co]
+endif::[]
+
+[[log-correlation]]
+== Log correlation
+
+Trace/log correlation can be set up in three different ways.
+
+[float]
+[[rails-tagged-logging]]
+==== Rails TaggedLogging
+
+Rails applications configured with an `ActiveSupport::TaggedLogging` logger can append the correlation IDs to log output.
+For example in your `config/environments/production.rb` file, add the following:
+
+[source,ruby]
+----
+config.log_tags = [ :request_id, proc { ElasticAPM.log_ids } ]
+
+# Logs will then include the correlation IDs:
+#
+# [transaction.id=c1ae84c8642891eb trace.id=b899fc7915e801b7558e336e4952bafe] Started GET "/" for 127.0.0.1 at 2019-09-16 11:28:46 +0200
+# [transaction.id=c1ae84c8642891eb trace.id=b899fc7915e801b7558e336e4952bafe] Processing by ApplicationController#index as HTML
+# [transaction.id=c1ae84c8642891eb trace.id=b899fc7915e801b7558e336e4952bafe]   Rendering text template
+# [transaction.id=c1ae84c8642891eb trace.id=b899fc7915e801b7558e336e4952bafe]   Rendered text template (Duration: 0.1ms | Allocations: 17)
+# [transaction.id=c1ae84c8642891eb trace.id=b899fc7915e801b7558e336e4952bafe] Completed 200 OK in 1ms (Views: 0.4ms | Allocations: 171)
+----
+**Note:** Because of the order in which Rails computes the tags for logs and executes the request, the span id might not be included.
```

Comment:
If you remove the bolding (and add a line above), this turns into an [admonition paragraph](https://github.com/elastic/docs#admonition-paragraphs), which I think is more useful here.
```suggestion

NOTE: Because of the order in which Rails computes the tags for logs and executes the request, the span id might not be included.
```

Reply:
I was following what is done in the [metrics](https://github.com/elastic/apm-agent-ruby/blob/master/docs/metrics.asciidoc) documentation but it seems more useful to have this note in an admonition paragraph. Thanks!